PR-Trigger/index.js

```diff
@@ -73,6 +75,16 @@ const httpTrigger = async function (context, _req) {
     } else if("issue" in webhook) {
         prNumber = webhook.issue.number
         prTitle = webhook.issue.title
+    } else if("sha" in webhook) {
+        // See https://github.com/maintainers/early-access-feedback/issues/114 for more context on getting a PR from a SHA
+        // TLDR: it's not in the API, and this search hack has been in used on Peril for the last ~3 years
+        const repoString = webhook.repository.full_name
+        const query = `${webhook.sha} type:pr is:open repo:${repoString}` 
+        const pr = await runQueryToGetPRMetadataForStatus(query)
+        if (pr) {
+            prNumber = pr.number
+            prTitle = pr.title
+        }

```

Comment:
basically go from a status hook to a PR number - https://github.com/artsy/peril-settings/blob/master/org/mergeOnGreen.ts#L22-L30